modules/ROOT/pages/update-an-api-configure-service-mesh-CLI.adoc

```diff
@@ -0,0 +1,86 @@
+= Update an API Using the CLI
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, updating
+
+APIs are constantly evolving. Anypoint Service Mesh allows you to update the specification of your API within the same major version (just minor and patch bumps allowed). You can update an API using Anypoint Platform or Anypoint Service Mesh:
+
+* <<update-api-AP, Update an API using Anypoint Platform>>
++
+Once a new version of an API is published to Exchange, the API instances associated with the asset can be updated to use this new version of the asset.
++
+Tags are updated by accesing the asset in Exchange or the API instance in API Manager, and modifying their tags through the corresponding UI.
+* <<update-api-cli,Update an API using Anypoint Service Mesh>>
++
+If the API resource exists in the cluster, it can be updated so it triggers changes in the Anypoint Platform, creating or modificating asset versions and API instances.
+
+
+[[update-api-AP]]
+== Update an API using Anypoint Platform
+
+=== Update API asset in Exchange
+To update an existing asset or create a new version of it using Anypoint Platform, perform the following steps:
+
+. Go to *Anypoint Platform > Exchange* and select the desired asset.
++
+The asset page is displayed. Now you can perform the following actions:
+. If you wish to upload a new version, click on *Add new version* on the right-side table.
+. If you wish to change the tags of a version, you can do it in the *Tags* section on the right-side panel.  
+
+=== Update API instance in API Manager
+To update an existing API instance, perform the following steps:
+
+. Go to *Anypoint Platform > API Manager*
+. Search and select your API instance in the table. 
++
+Once inside the API details page, you can perform the following actions:
+. Update the instance tags, at the *API Configuration* section.
+. Update the asset version it points to.
+image::update-api-asset.png[Display asset update]
+
+
+[[update-api-cli]]
```

Comment:
One last, comment,  I would reorder and put the update of service mesh before the updates on the platform.

Reply:
Done! Thanks!